wp-content/themes/twentytwenty-child/style.css

```diff
@@ -11,19 +11,52 @@ Version: 1.0
 /** VARIABLES **/
 /***************/
 :root {
-  --darker-blue: #0a5387;
-  --dark-blue: #29588C;
-  --light-blue: #4CB4E7;
+  /* Colors */
+  --darker-blue:  #0a5387;
+  --dark-blue:    #29588C;
+  --light-blue:   #4CB4E7;
 
+  --lighter-gray: #F4F4F4;
+  --light-gray:   #EEEEEE;
+  --medium-gray:  #CCCCCC;
+
+  --dark-red : #c62828;
+
+  /* Gradients */
   --header-gradient: linear-gradient(
           90deg,
           rgba(10,83,135,1) 0%,
-          rgba(25,95,145,1) 35%,
-          rgba(80,119,161,1) 55%,
-          rgba(255,255,255,1) 100%
+          rgba(25,95,145,1) 47%,
+          rgba(80,119,161,1) 79%,
+          rgba(138,164,192,1) 100%
   );
 
+  /* Borders */
+  --thin-light-gray:  1px solid var(--light-gray);
```

Comment:
Any chance to integrate SASS? :)

--- Nice use of variables for global colors! :+1:

Reply:
I think SASS will be an excellent idea! A lot of the selectors could and _should_ be nested. The structure will be a lot easier to follow too.